app/templates/main/index.html

```diff
@@ -119,6 +119,84 @@
       </div>
     </div>
 
+    <!-- HANDLEBARS.JS TEMPLATES COMPILED IN MAP.JS -->
+    <!-- Populate location search marker info-->
+    <script id="locationmarker-template" type="text/x-handlebars-template">
+      <div class="loc-marker-info">
+          <h3 class="ui header">
+              {% raw %}{{ name }}{% endraw %}
```

Comment:
No, you need the `{% raw %}{% endraw %}` to avoid it from being parsed by Jinja, Jinja still tries to parse `{{{ name }}}`